base/inference.jl

```diff
@@ -3359,7 +3360,8 @@ function is_pure_builtin(f::ANY)
              f === Intrinsics.checked_srem_int ||
              f === Intrinsics.checked_urem_int ||
              f === Intrinsics.check_top_bit ||
-             f === Intrinsics.sqrt_llvm)
+             f === Intrinsics.sqrt_llvm ||
+             f === Intrinsics.cglobal)  # cglobal throws an error for symbol-not-found
```

Comment:
True... it's just the BLAS code relies on a `cglobal` call in void context throwing an error, which is a very atypical use of an intrinsic.